server/server_test.go

```diff
@@ -1447,7 +1447,7 @@ func TestCacheControlHeaders(t *testing.T) {
 			filename:                    "main.e4188e5adc97bbfc00c0.js",
 			createFile:                  false,
 			expectedStatus:              404,
-			expectedCacheControlHeaders: []string{"no-cache"},
+			expectedCacheControlHeaders: nil,
```

Comment:
This is caused by go internal changes. https://github.com/golang/go/issues/66343
Leaving the code unmodified to guarantee that no-cache will be there if golang decide to revert the change the internal behavior. Leaving it there is a no-op.